egs2/fsc_challenge/slu1/local/score.sh

```diff
@@ -0,0 +1,53 @@
+#!/usr/bin/env bash
```

Comment:
Given that we have a SLU-specified template, do you think it would be good to put the score.sh into `scripts/utils` and make it standard?

When I was using the SLU recipe, I found several times that, if just calling run.sh, usually there is no option to use some specific parameters (e.g., we will fix the 5best, but what if instead 5best, we want 10best). We need to open the score.sh -> understand the logic --> find corresponding directory --> call the score.sh again. This is sometimes confusing for newcomers.

It does not have to be settled in this PR. If this PR is in a hurry, we can firstly finish this one and leave that in the future.

Reply:
I think this is a great point Jiatong! I agree that the current setup is confusing. I believe in slu.sh, this has been handled since it looks at the inference folder from the script itself (https://github.com/espnet/espnet/blob/a7bd6522b32ec6472c13f6a2289dcdff4a846c12/egs2/TEMPLATE/slu1/slu.sh#L1497). I will also ensure in a future PR that for all previous SLU recipes that use asr.sh also get inference folder from script itself so that score.sh does not need to be run seperately.

I have noted this and will open another PR after Interspeech for this. I will hopefully like to merge this PR before I leave for Interspeech.

Reply:
Sounds great! Thanks for letting me know the status.

Reply:
I can merge it after the CI test is fixed~